moped-editor/src/views/projects/projectView/ProjectSummary/TagsSection.js

```diff
@@ -160,7 +134,24 @@ const TagsSection = ({ projectId, handleSnackbar }) => {
             Add tag
           </Button>
         </Toolbar>
-        <Box className={classes.chipContainer}>
+        <Box
+          sx={{
+            display: "flex",
+            justifyContent: "left",
+            flexWrap: "wrap",
+            listStyle: "none",
+            padding: (theme) => {
+              {
+                theme.spacing(1);
+              }
+              {
+                theme.spacing(2);
+              }
```

Comment:
here is what is blocking the build

Reply:
my apologies, y'all! i fixed the build and made some tweaks so the the tags section aligns perfectly with the subproject section both vertically and horizontally. there are some add values but it works!

<img width="522" height="411" alt="Screenshot 2025-08-12 at 3 54 42 PM" src="https://github.com/user-attachments/assets/7bde74d7-9a23-44b9-be87-3ad2e5f981cf" />